uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -89,19 +90,40 @@ Control {
                 orientation: Qt.Horizontal
             }
 
-            Repeater {
-                id: repeater
-                property int downItem
-
-                model: controller.levels
-                delegate: ToolButton {
-                    visible: !closer.checked
-                    checked: controller.selectedLevelId === model.modelId
-                    autoExclusive: true
-                    Layout.fillWidth: true
-                    text: collapser.checked ? model.shortName : model.longName
-                    onClicked: {
-                        controller.selectedLevelId = model.modelId
+            Flickable {
+                visible: !closer.checked
+
+                // dont need to use the id of the column
+                contentHeight: contentItem.childrenRect.height
+                Layout.fillWidth: true
+                height: repeater.buttonHeight * maxNumberLevels
+                clip: true
+                ScrollBar.vertical: ScrollBar {}
+
+                Column {
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    Component.onCompleted: console.log("width col", width)
+                    Repeater {
+                        id: repeater
+
+                        property int downItem
+                        property int buttonHeight: 50
+
+                        model: controller.levels
+                        delegate: ToolButton {
+                            Component.onCompleted: this.height = repeater.buttonHeight
```

Comment:
not creating a binding `height : repeater.buttonHeight` because the property `buttonHeight` will not change, so it would be a waste of a binding creation